config/index.js

```diff
@@ -432,6 +432,20 @@ var conf = convict({
       default: '4 weeks'
     }
   },
+  tokenPruning: {
+    enabled: {
+      doc: 'Turn on Redis token pruning',
+      format: Boolean,
+      default: true,
+      env: 'TOKEN_PRUNING_ENABLED'
+    },
+    maxAge: {
+      doc: 'Age at which to prune expired tokens from Redis',
+      format: 'duration',
+      default: '1 month',
```

Comment:
May be worth pointing out that this acts as a secondary gating, really. Only tokens that are already expired reach the point of checking against this.

Perhaps we could even drop this setting entirely and just lean on the existing token-expiry config?